src/rastervision/crs_transformers/rasterio_crs_transformer.py

```diff
@@ -4,22 +4,48 @@
 
 
 class RasterioCRSTransformer(CRSTransformer):
+    """Transformer for a RasterioRasterSource.
+
+    This assumes that the map coordinates are always in lon/lat format.
+    """
+
     def __init__(self, image_dataset):
+        """Construct transformer.
+
+        Args:
+            image_dataset: Rasterio DatasetReader
+        """
         self.image_dataset = image_dataset
-        self.web_proj = pyproj.Proj(init='epsg:4326')
+        self.map_proj = pyproj.Proj(init='epsg:4326')
         image_crs = image_dataset.crs['init']
         self.image_proj = pyproj.Proj(init=image_crs)
 
-    def web_to_pixel(self, web_point):
+    def map_to_pixel(self, map_point):
+        """Transform point from map to pixel-based coordinates.
+
+        Args:
+            map_point: (long, lat) tuple
+
+        Returns:
+            (x, y) tuple in pixel coordinates
```

Comment:
In GeoTrellis we differentiate between grid coordinates (pixel coordinates) and map coordinates  by always referring to the former as `(col, row)` and always referring to  the latter as `(x, y)`. We might want to adopt a similar strategy - it  has been useful for (somewhat effectively) preventing bugs that are based on folk not knowing in which reference  system they are working in.

Reply:
l will consider doing this when I write unit tests (and lightly refactor) the Labels and LabelStore classes.